server/graphql/schema.graphql

```diff
@@ -0,0 +1,129 @@
+type BookReview {
+  id: ID!
+  review: String
+  rating: Int!
+  reviewer_email: String!
+  created_at: String!
+  book: Book!
+}
+
+type Book {
+  id: ID!
+  title: String!
+  description: String!
+  published_date: String!
+  author: Author!
+  reviews: [BookReview!]
+  avg_rating: Float!
+  total_reviews: Int!
+}
+
+type AuthorReview {
+  id: ID!
+  review: String
+  rating: Int!
+  reviewer_email: String!
+  created_at: String!
+  author: Author!
+}
+
+type Author {
+  id: ID!
+  name: String!
+  biography: String!
+  born_date: String!
+  books: [Book!]
+  reviews: [AuthorReview!]
+  avg_rating: Float!
+  total_reviews: Int!
+}
+
+input BookFilter {
```

Comment:
Can we have seperate files for different entities, each encapsulating `typeDefs` for different Entities and all these files are inside a folder called `typeDefs`.

Reply:
done segregated resolvers and typedefs